### PR TITLE
feat(tooltip): adding tooltip component

### PR DESCRIPTION
**Pull request by ghost**
added custom styling, functionality and story for tooltip

### CODE REVIEW

File: src/components/STooltip.vue [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+</script>
+<style lang="scss" scoped>
+  .tooltip,
+  .callout {
+    padding: 24px;
+    background-color: white;
+    color: var(--v-primary-base);
+    opacity: 1 !important;
+    box-shadow: rgba(0, 0, 0, 0.3) 4px 4px 4px ;
+    border: 1px solid #ccc;
```
Comment (aldarund) [inline src/components/STooltip.vue]: we have color border

File: src/components/STooltip.vue [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    background-color: white;
+    position: absolute;
+    -moz-transform: rotate(-45deg);
+    -webkit-transform: rotate(-45deg);
+  }
+  .left:after {
+    left: -11px;
+    bottom: 40%;
+    border-left: 1px solid #bbb;
+    border-top: 1px solid #bbb;
```
Comment (aldarund) [inline src/components/STooltip.vue]: maybe anotehr color defined if its gonna to be used in other components

File: src/components/STooltip.vue [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        {{ text }}
+      </SBtn>
+    </template>
+    <slot />
+  </VTooltip>
+</template>
+<script lang="ts">
+import { Component, Prop, Vue } from 'nuxt-property-decorator'
+import { mdiHelpCircleOutline } from '@mdi/js'
+import SBtn from '../../src/components/SBtn.vue'
```
Comment (aldarund) [inline src/components/STooltip.vue]: i`m not sure if this will work e.g. path like this when used with main project. Probably you need to start with './

**Review by aldarund — APPROVED** [review]

File: src/components/STooltip.vue (hunk 3 above)
Comment (ghost) [reply src/components/STooltip.vue]: updated

**Pull request merged**